notebooks/notebooks/iclr2022/estimate-flops-adv.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.                + MACS_DICT[victim_net_size] * adv_moves / 2
</pre><p>why are we multiplying by adv_moves here again such that, after expansion, this term is <code>MACS_DICT[victim_net_size] * adv_moves**2</code> ? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Good catch. This was a typo on my part. Fixing this has shifted our compute estimates back up (see updated numbers in the first PR comment).